demo.md

```diff
@@ -84,6 +84,9 @@ Login to the Minio UI using a web browser (port 9000) after exposing obtaining t
 ```
 kubectl get service argo-artifacts -o wide
 ```
+A recent version of the Minio UI uses `argo-artifacts-minio` as service name, instead of `argo-artifacts`. You can check this by running
+`kubectl get services` and inspecting the output.
+
```

Comment:
Actually if this is the case, we should just change the guide to use the current version of minio.